src/odemis/acq/align/test/spot_test.py

```diff
@@ -290,6 +292,32 @@ def test_find_grid_with_shear(self):
         self.assertNotEqual(shear, 0)
         self.assertNotEqual(more_shear, 0)
 
+    def test_find_grid_with_rotation(self):
+        """
+        Create an image with a grid of 8 by 8 spots. Then rotate the spots and test if the correct rotation is found.
+        """
+        # test for a rotated image
+        image = numpy.zeros((256, 256))
+        image[54:150:12, 54:150:12] = 1
+        # get the spot indices
+        indices = numpy.where(image)
+        indices = numpy.column_stack((indices[0], indices[1]))
+        img_rotation = math.radians(10)
+        transform = ScalingTransform(rotation=img_rotation)
+        # get the rotated indices
+        rotated_indices = transform.apply(indices)
+        rotated_indices -= numpy.min(rotated_indices)
+        rotated_indices += 10  # make sure each spot is at least 10 pixels away from the border
+        max_idx = int(numpy.max(rotated_indices))
+        rotated_image = numpy.zeros((max_idx + 10, max_idx + 10))
```

Comment:
```suggestion
        max_idx = int(numpy.max(rotated_indices))
        max_idx += 10
        rotated_image = numpy.zeros((max_idx, max_idx))
```

Reply:
Then, just write `max_idx = int(numpy.max(rotated_indices)) + 10`
But maybe max_idx is not a good name anymore?
I think the original code was pretty fine.

Reply:
I'll leave it like it was